docs/recipes/reducers/07-UpdatingNormalizedData.md

```diff
@@ -0,0 +1,304 @@
+# Managing Normalized Data
+
+As mentioned in [Normalizing State Shape](./06-NormalizingStateShape.md), the Normalizr library is frequently used to transform nested response data into a normalized shape suitable for integration into the store.  However, that doesn't address the issue of executing further updates to that normalized data as it's being used elsewhere in the application.  There are a variety of different approaches that you can use, based on your own preference.  We'll use the example of adding a new Comment to a Post.
+
+## Standard Approaches
+
+### Simple Merging
+
+One approach is to simply merge the contents of the action in to the existing state:
+
+```js
+import merge from "lodash/object/merge";
+
+function commentsById(state = {}, action) {
+    default : {
+       if(action.entities && action.entities.comments) {
+           return merge({}, state, action.entities.comments.byId);
```

Comment:
I think I borrowed this example directly from one of Dan's comments in an issue thread.  The idea is that you're not _replacing_ the existing state, you're updating it.  Not sure if there's a better way to illustrate that.